File: codegen-result-example/src/typescript/api/evt/BaseQueryEvt.ts
    import {BaseEvt} from "./BaseEvt";

/**
 * 统一的查询对象
**/

export interface  BaseQueryEvt extends BaseEvt {

        /**
            *属性说明：查询大小，示例输入：
            *在java中的类型为：Integer
        **/
        querySize?: number;
        /**
            *属性说明：查询页码，示例输入：
            *在java中的类型为：Integer
        **/
        queryPage?: number;
}

File: examples/swagger-3/src/test/resources/asserts/java/spring_cloud_openfeign/src/com/wuxp/codegen/swagger3/evt/BaseQueryEvt.java
package com.wuxp.codegen.swagger3.evt;

import lombok.Data;
import lombok.experimental.Accessors;
import javax.validation.constraints.*;
        import com.wuxp.codegen.swagger3.evt.BaseEvt;

@Data
@Accessors(chain = true)
public class  BaseQueryEvt extends BaseEvt {

            /**
                *字段在java中的类型为：Integer
            **/
         Integer querySize;

            /**
                *字段在java中的类型为：Integer
            **/
         Integer queryPage;

}
